src/sources/mssql/mssql.lisp

```diff
@@ -69,17 +69,28 @@
                      :excluding excluding)
 
       ;; fetch view (and their columns) metadata, covering comments too
-      (let* ((view-names (unless (eq :all materialize-views)
-                           (mapcar #'matview-source-name materialize-views)))
-             (including
-              (loop :for (schema-name . view-name) :in view-names
-                 :do (let* ((schema-name (or schema-name "dbo"))
-                            (schema-entry
-                             (or (assoc schema-name including :test #'string=)
-                                 (progn (push (cons schema-name nil) including)
-                                        (assoc schema-name including
-                                               :test #'string=)))))
-                       (push-to-end view-name (cdr schema-entry))))))
+      (let* (
+          (view-names
+            (progn
+              (let
+                ((names (unless
+                  (eq :all materialize-views)
+                  (mapcar #'matview-source-name materialize-views))))
+                ;; Debugging statement
+                ;; Return the computed value for the let* binding
+                names)))
+          (including nil)
+        )
+        (loop :for (schema-name . view-name) :in view-names
+          :do (let* (
+              (schema-name (or schema-name "dbo"))
+              (schema-entry (or
+                (assoc schema-name including :test #'string=)
+                (let
+                  ((new-entry (cons schema-name nil))); Initially nil, intending to be a list
+                  (push new-entry including)
+                  new-entry))))
+          (push-to-end view-name (cdr schema-entry))))
```

Comment:
What I mean is: completely clean up the debugging alterations, and this is the indentation that should appear here:

```
(let* ((view-names (unless (eq :all materialize-views)
                     (mapcar #'matview-source-name materialize-views)))
       (including nil))
  (loop :for (schema-name . view-name) :in view-names
        :do (let* ((schema-name (or schema-name "dbo"))
                   (schema-entry (or
                                  (assoc schema-name including :test #'string=)
                                  (let ((new-entry (cons schema-name nil)))
                                    (push new-entry including)
                                    new-entry))))
              (push-to-end view-name (cdr schema-entry))))
  …)
```

I don't fully understand what's changed here yet, though